src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java

```diff
@@ -59,14 +63,16 @@ private void readObject(java.io.ObjectInputStream inputStream) {
     }
 
     /**
-     * {@link} // violation
+     * {@link} // violation 'Javadoc comment at column 13 has parse error. Details: mismatched
+     * input '}' expecting {LEADING_ASTERISK, WS, NEWLINE}" while parsing JAVADOC_INLINE_TAG'
      */
     public void method7() {
 
     }
 
     /**
-     * {@linkplain } // violation
+     * {@linkplain } // violation 'Javadoc comment at column 19 has parse error. Details: no viable
```

Comment:
All `// violation` shout be outside of multiline comment `/**`

Reply:
@sd9616 , did you finish?

Reply:
Yes, it is done